docs/understand/file_reorg.md

```diff
@@ -42,14 +44,13 @@ distributions. Following is the ROCm proposed file structure.
               | -- architecture independent misc files
 ```
 
-## Changes from earlier ROCm versions
+## Changes From Earlier ROCm Versions
 
-ROCm with the file reorganization is going to have a lean structure. Following
-table gives the comparison with new and old folder structure.
+The following table provides a brief overview of the new ROCm FHS layout, compared to the layout of earlier ROCm versions.
 
 ```none
  ______________________________________________________
-|  New File Structure         |  Old File Structure    |
+|  New ROCm Layout            |  Previous ROCm Layout  |
 |_____________________________|________________________|
 | /opt/rocm-<ver>             | /opt/rocm-<ver>        |
```

Comment:
instead of /opt/ we should state the rocm-installation-path which is by default /opt/